main/src/cgeo/geocaching/connector/oc/OCApiConnector.java

```diff
@@ -108,4 +109,24 @@ public int getTokenSecretPrefKeyId() {
     public boolean isSearchForMyCaches(final String username) {
         return false;
     }
+
+    public boolean supportsPersonalNote() {
+        if (this.getApiSupport() == ApiSupport.current) {
```

Comment:
In addition to the Api-level, the user also needs to be authenticated on ```level3``` in order to have the possibility to upload a note (i.e. he needs to be logged in).
While in the gc.com version, this is assumed (you really cannot do anything useful without a login) this is not necessary the case for opencaching. Even without authentication you can download full cache details and go caching.
I suggest to move this change to the ```OCapiLiveConnector``` which handles level3-authenticated communication.